docs/user-manuals/jobsidecarterminator.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Job Sidecar Terminator
+---
+
+**FEATURE STATE:** Kruise v1.4.0
+
+In Kubernetes world, we hope that Pods controlled by job-type workload enter `Completed` phase immediately once the task executed in main container is completed.
+However, in the scene where the these Pods have long-running sidecar containers, Pod may be out of our expectations because the long-running sidecar container cannot be terminated when the main containers completed.
```

Comment:
In scenarios where Pods include sidecar containers, a common challenge arises due to sidecar containers cannot exit after the main container has exited, the job will remain incomplete ?